model/Core/Vocabularies/RelationshipType.md

```diff
@@ -12,73 +12,79 @@ Provides information about the relationship between two Elements.
 For example, you can represent a relationship between two different Files,
 between a Package and a File, between two Packages, or between one SPDXDocument and another SPDXDocument.
 
+Relationship names be descriptive enough to easily deduce the correct direction
+from their name. The best way to do this is to make sure that the relationship
+name completes the sentence:
+
+`from` (is) (a) `RELATIONSHIP` `to`
+
 ## Metadata
 
 - name: RelationshipType
 
 ## Entries
 
-- affects: (Security/VEX) Designates one or more elements as affected by a vulnerability
-- amends: Every `to` Element amends the `from` Element
-- ancestor: Every `to` Element is an ancestor of the `from` Element
-- availableFrom:  This relationship is used to identify additional suppliers where an artifact is available from.
-- buildDependency: Every `to` Element is a build dependency of the `from` Element
-- buildTool: Build tool used to build an Element. This may be used to describe the build tool of a Build instance
+- affects: (Security/VEX) The `from` vulnerability affect each `to` Element
+- amendedBy: The `from` Element is amended by each `to` Element
+- hasAncestor: The `from` Element has ancestor described by each `to` Element
+- availableFrom: The `from` Element is available from the additional supplier described by each `to` Element
+- hasBuildDependencyOn: The `from` element has a build dependency on each `to` Element
+- usesBuildTool: The `from` Build element uses each `to` element as a build tool
```

Comment:
Suggest" The `from` Element uses each `to` Element as a build tool" will align more with the model.